stdlib/public/core/EmptyCollection.swift

```diff
@@ -51,6 +51,12 @@ public struct EmptyCollection<Element> : Collection {
     return 0
```

Comment:
And even MutableCollection.  There are no elements, so there is nothing to mutate.